### PR TITLE
doc: add meeting notes of 2020-08-06

### PR DESCRIPTION
**Pull request by Trott**
Closes: https://github.com/nodejs/TSC/issues/900

### CODE REVIEW

**Comment by Trott** [comment]
@nodejs/tsc

File: meetings/2020-08-06.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  * GitHub is still sorting out the details of forwarding for git clients and what will and won’t be supported.
+  * Myles had proposed doing this on a non-core repo (nodejs.dev and/or citgm). Myles opened issues on those repos. Myles will see about getting this moving in those repos.
+  * Anatoli and Myles discuss the importance of clear messaging around the lifecycle of the transition.
+
+
+### nodejs/TSC
+
+* Resigning from CPC Voting Member [#898](https://github.com/nodejs/TSC/issues/898)
+
```
Comment (mmarchini) [inline meetings/2020-08-06.md:47]:
```suggestion
```

File: meetings/2020-08-06.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  * Matteo will put a comment in the issue calling for people to volunteer/self-nominate over the next two weeks. 
+
+### nodejs/admin
+
+* Audit Google account access [#389](https://github.com/nodejs/admin/issues/389)
+
```
Comment (mmarchini) [inline meetings/2020-08-06.md:53]:
```suggestion
```

File: meetings/2020-08-06.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+### CPC and Board Meeting Updates
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* Rename default branch from "master" to "main" or something similar [#33864](https://github.com/nodejs/node/issues/33864)
+
```
Comment (mmarchini) [inline meetings/2020-08-06.md:38]:
```suggestion
```

**Pull request merged**